code/modules/power/turbine/turbine.dm

```diff
@@ -117,9 +117,17 @@
 	return ..()
 
 /obj/machinery/power/turbine/crowbar_act_secondary(mob/living/user, obj/item/tool)
+	if(!panel_open)
+		balloon_alert(user, "panel is closed!")
+		return TOOL_ACT_TOOLTYPE_SUCCESS
```

Comment:
just `return` here (and every other place except the last one, since it isn't actually successful